regression/contracts/assigns_enforce_structs_07/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--enforce-all-contracts _ --pointer-check
+^EXIT=10$
+^SIGNAL=0$
+^\[f1.\d+\] line \d+ Check that p\-\>buf\[\(.*\)0\] is assignable\: FAILURE$
+^\[f1.pointer\_dereference.\d+\] line \d+ dereference failure\: pointer NULL in p\-\>buf\: FAILURE$
+^\[f2.\d+\] line \d+ Check that pp\-\>p\-\>buf\[\(.*\)0\] is assignable\: FAILURE$
+^\[f2.pointer\_dereference.\d+\] line \d+ dereference failure\: pointer NULL in pp\-\>p\-\>buf\: FAILURE$
```

Comment:
[nit]

We could reduce some of the escaping here. In particular, `-`, `>` and `:` need not be escaped here (and elsewhere, in other `.desc` files). Not a big issues, but just to makes these a little more readable

Reply:
I removed all instances of these escaping partners from our contracts regression tests.

Reply:
Thanks.